# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../shadow/no-wheel-su.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The now, enabled by default, wheel group is annoying for most users,
we comment it out, experts can re-enable it.

  - Rene Rebe <rene@exactcode.de>

diff -urN shadow-4.0.14/etc/pam.d/su shadow-4.0.14-fixed/etc/pam.d/su
--- shadow-4.0.14/etc/pam.d/su	2006-01-02 21:40:28.000000000 +0100
+++ shadow-4.0.14-fixed/etc/pam.d/su	2006-06-06 18:03:00.000000000 +0200
@@ -3,7 +3,7 @@
 # Uncomment the following line to implicitly trust users in the "wheel" group.
 #auth		sufficient	pam_wheel.so trust use_uid
 # Uncomment the following line to require a user to be in the "wheel" group.
-auth		required	pam_wheel.so use_uid
+#auth		required	pam_wheel.so use_uid
 auth		include		system-auth
 account		include		system-auth
 password	include		system-auth
